packages/shared/src/helpers/https.ts

```diff
@@ -25,11 +25,11 @@ interface MakeRequest {
 class Http {
   private _baseUrl: string;
 
-  private _auth: IAuthHelper | null;
+  private _storageService: IStorageService | null;
 
-  constructor(baseUrl: string, auth?: IAuthHelper) {
+  constructor(baseUrl: string, storageService?: IStorageService) {
     this._baseUrl = baseUrl;
-    this._auth = auth;
+    this._storageService = storageService || null;
```

Comment:
Storage and Auth are two different entities, it's not correct to tight it in one God like entity, please add third service as Storage